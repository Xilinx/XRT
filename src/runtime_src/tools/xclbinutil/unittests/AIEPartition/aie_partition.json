{
	"aie_partition": {
		"partition": {
			"column_width": 1,
			"start_columns": [1, 2, 3, 4]
		},
		"PDIs": [{
				"uuid": "00000000-0000-0000-0000-000000001111",
				"file_name": "./1111.txt",
				"cdo_groups": [{
						"name": "DPU",
						"type": "PRIMARY",
						"pdi_id": "0xF0",
						"dpu_kernel_id": "0x100",
						"pre_cdo_groups": ["0xC0"]
					},
					{
						"name": "PP0",
						"type": "LITE",
						"pdi_id": "0xF1",
						"dpu_kernel_id": "0x101",
						"pre_cdo_groups": ["0xC0"]

					},
					{
						"name": "PP1",
						"type": "LITE",
						"pdi_id": "0xF2",
						"dpu_kernel_id": "0x102",
						"pre_cdo_groups": ["0xC0"]
					},
					{
						"name": "shared",
						"pdi_id": "0xC0",
						"pre_cdo_groups": ["0xC1"]
					},
					{
						"name": "routing",
						"pdi_id": "0xC1"
					}
				]
			},
			{
				"uuid": "00000000-0000-0000-0000-000000002222",
				"file_name": "./2222.txt",
				"cdo_groups": [{
					"name": "PP3",
					"type": "LITE",
					"pdi_id": "0xF1",
					"dpu_kernel_id": "0x104"
				}]
			}
		]
	}
}
