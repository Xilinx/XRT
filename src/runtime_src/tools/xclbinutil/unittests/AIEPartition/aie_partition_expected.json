{
    "aie_partition": {
        "name": "Flavor",
        "operations_per_cycle": "135",
        "partition": {
            "column_width": "1",
            "start_columns": [
                "1",
                "2",
                "3",
                "4"
            ]
        },
        "PDIs": [
            {
                "uuid": "00000000-0000-0000-0000-000000001111",
                "file_name": "00000000-0000-0000-0000-000000001111.pdi",
                "cdo_groups": [
                    {
                        "name": "DPU",
                        "type": "PRIMARY",
                        "pdi_id": "0xf0",
                        "dpu_kernel_ids": [
                            "0x100"
                        ],
                        "pre_cdo_groups": [
                            "0xc0"
                        ]
                    },
                    {
                        "name": "PP0",
                        "type": "LITE",
                        "pdi_id": "0xf1",
                        "dpu_kernel_ids": [
                            "0x101"
                        ],
                        "pre_cdo_groups": [
                            "0xc0"
                        ]
                    },
                    {
                        "name": "PP1",
                        "type": "LITE",
                        "pdi_id": "0xf2",
                        "dpu_kernel_ids": [
                            "0x102"
                        ],
                        "pre_cdo_groups": [
                            "0xc0"
                        ]
                    },
                    {
                        "name": "shared",
                        "pdi_id": "0xc0",
                        "pre_cdo_groups": [
                            "0xc1"
                        ]
                    },
                    {
                        "name": "routing",
                        "pdi_id": "0xc1"
                    }
                ]
            },
            {
                "uuid": "00000000-0000-0000-0000-000000002222",
                "file_name": "00000000-0000-0000-0000-000000002222.pdi",
                "cdo_groups": [
                    {
                        "name": "PP3",
                        "type": "LITE",
                        "pdi_id": "0xf1",
                        "dpu_kernel_ids": [
                            "0x104",
                            "0x105"
                        ]
                    }
                ]
            }
        ]
    }
}
